.github/workflows/main_workflow.yml

```diff
@@ -92,6 +92,9 @@ jobs:
     - uses: actions/setup-python@master
       with:
         python-version: ${{ matrix.python-version }}
+    - uses: actions/setup-go@v3
+      with:
+        go-version: "1.17.7"
```

Comment:
```
GO111MODULE=on GOPATH=~/go go get -u -v github.com/yoheimuta/protolint/cmd/protolint@v0.27.0
go: go.mod file not found in current directory or any parent directory.
	'go get' is no longer supported outside a module.
	To build and install a command, use 'go install' with a version,
	like 'go install example.com/cmd@latest'
	For more information, see https://golang.org/doc/go-get-install-deprecation
	or run 'go help get' or 'go help install'.
```
https://github.com/valory-xyz/open-autonomy/blob/8de9e2d0c93070bbcfe6270e4ee3aec7037cc36a/Makefile#L196-L197

pinned 1.17.7 because that is what we use in the rest of CI
https://github.com/valory-xyz/open-autonomy/blob/8de9e2d0c93070bbcfe6270e4ee3aec7037cc36a/.github/workflows/main_workflow.yml#L155
https://github.com/valory-xyz/open-autonomy/blob/8de9e2d0c93070bbcfe6270e4ee3aec7037cc36a/.github/workflows/main_workflow.yml#L187